rom/dev/src/flow/cold_reset/fw_processor.rs

```diff
@@ -286,6 +286,12 @@ impl FirmwareProcessor {
                         resp.populate_chksum()?;
                         txn.send_response(resp.as_bytes())?;
                     }
+                    #[cfg(feature = "test_only_commands")]
+                    CommandId::TEST_ONLY_TRIGGER_CPU_FAULT => {
+                        // the print is necessary to ensure the compiler doesn't optimize out the unsafe code below
+                        cprintln!("Unsafe write to null mutable pointer");
```

Comment:
I still don't really understand why this is getting optimized out. Not sure if @korran would have any insight here.

Inline asm should ensure it doesn't get optimized out though.

Reply:
As I mentioned offline, I think we should do something like what I proposed at https://github.com/chipsalliance/caliptra-sw/issues/1035

Reply:
Done